api/src/main/java/com/nicknathanjustin/streamercontracts/donations/OpenDonationDto.java

```diff
@@ -0,0 +1,30 @@
+package com.nicknathanjustin.streamercontracts.donations;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import java.math.BigDecimal;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class OpenDonationDto {
+    private UUID donationId;
```

Comment:
Probably don't need to return the donation ID right?

Reply:
I think we will need it to identify which donation action is being taken on